spec/categorical_spec.rb

```diff
@@ -0,0 +1,37 @@
+describe Daru::Vector do
+  context "initialize" do
+    let(:dv) { Daru::Vector.new [:a, 1, :a, 1, :c], type: :category }
+    subject { dv }
+    
+    it { is_expected.to be_a Daru::Vector }
+    its(:size) { is_expected.to eq 5 }
+    its(:type) { is_expected.to eq :category }
+    its(:ordered?) { is_expected.to eq false }
+    its(:to_a) { is_expected.to eq [:a, 1, :a, 1, :c] }
+  end
+  
+  context "#to_factor!" do
```

Comment:
I know some statistic software like R uses this as a synonym of categories, but is it transparent for everyone?.. Like, you have two absolutely different words for the same thing....